Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,25 @@ def self._parse spec
     super
   end
 end
+
+class BottlePkgVersion < PkgVersion
```

Comment:
Seems it should maybe have a better name then and BottleVersion be folded into another class.